torch_xla/csrc/aten_xla_type.h

```diff
@@ -827,6 +827,10 @@ class AtenXlaType {
   static at::Tensor& resize_(at::Tensor& self, at::IntArrayRef size,
                              c10::optional<at::MemoryFormat> memory_format);
 
+  static at::Tensor round(const at::Tensor& self);
```

Comment:
Could you also add `round_out ` to https://github.com/pytorch/xla/blob/master/scripts/gen.py#L100? This way the out function will be redirect to use the xla implementation here. Example in https://github.com/pytorch/xla/pull/1592/files

Reply:
Done, thanks.